src/javascripts/components/planes.js

```diff
@@ -8,9 +8,9 @@ const showPlanes = (arr) => {
     <img class="card-img-top" src=${item.planeImage} alt=${item.planeMake}">
     <div class="card-body">
     <h5 class="card-title">${item.planeMake}</h5>
-    <p class="card-text bold">${item.s ? `<span class="badge badge-info sale-badge"><i class="fa fa-bell" aria-hidden="true"></i> Plane Model</span> Plane Model ${item.planeModel}` : `Plane Model ${item.planeModel}`}</p>
+    <p class="card-text bold">${item.s ? `<span class="badge badge-info sale-badge"><i class="fa fa-bell" aria-hidden="true"></i> Plane Model</span> Plane Model ${item.planeModel}` : `Plane Model: ${item.planeModel}`}</p>
```

Comment:
What is this line doing? Is there ever a case where the badge shows up?

Reply:
That's my fault! I was looking at my Almost Amazon for reference and I mindlessly added that